Demos/FluentUIDemo_iOS/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -520,6 +522,7 @@
 		A5CEC23220E452B80016922A /* Demos */ = {
 			isa = PBXGroup;
 			children = (
+				6F4A92EE2DD268A600B4B136 /* PillButtonDemoController_SwiftUI.swift */,
```

Comment:
nit: alphabetize, I believe this controls the order they show up in the file navigator in xcode